bindings/wysiwyg-ffi/src/ffi_composer_model.rs

```diff
@@ -220,7 +220,10 @@ impl ComposerModel {
         let link = Utf16String::from_str(&link);
         let text = Utf16String::from_str(&text);
         Arc::new(ComposerUpdate::from(
-            self.inner.lock().unwrap().set_link_with_text(link, text),
+            self.inner
+                .lock()
+                .unwrap()
+                .set_link_with_text(link, text, None),
```

Comment:
Following my comment below, we should probably expose a new Enum for the `data-mention-type` in order for the client to give the composer knowledge about what kind of link it is trying to create.

Reply:
I agree, but think that work will fit better with implementing the ability to pass in the mention type when calling the function